forte/pipeline.py

```diff
@@ -281,24 +323,89 @@ def _dump_to_config(self):
         Returns:
             dict: A dictionary storing IR.
         """
-        configs: List[Dict] = []
-        configs.append(
+
+        def get_type(instance) -> str:
```

Comment:
We have some functions like this here:

https://github.com/asyml/forte/blob/master/forte/utils/utils.py#L35